### ISSUE TITLE
accessibility issue 30 -35 (medium) - put pagination buttoms at botton in nav element

### ISSUE THREAD

**Issue by yulgit1** [ycba-cia/blacklight-collections2#250]
The “list” and “masonry” links should be buttons, with an aria-pressed attribute (for a toggle button).  ARIA Authoring Practices for toggle buttons:
https://www.w3.org/TR/wai-aria-practices-1.1/#button

**Comment by yulgit1** [comment]
addressed: https://github.com/ycba-cia/blacklight-collections2/commit/08f68ecbac0e8b0eea88b7d45222fd9fe4fcba8a
